.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up shell environment
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y shellcheck
+        shell: bash
+      - name: Run shellcheck
+        run: shellcheck --severity warning scripts/upgrade.sh scripts/tests.sh
+
+  test:
+    name: Run Unit Tests
+    runs-on: ubuntu-latest
+    container:
+      image: busybox
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Run unit tests
+        run: |
+          cd scripts
+          chmod +x ./tests.sh
```

Comment:
just `chmod a+x scripts/tests.sh; git add scripts/test.sh` in your git repo and then check in that change.
```suggestion
```